frontend/src/components/forms/SignupForm.module.css

```diff
@@ -3,7 +3,14 @@
 }
 
 .signup_button {
-    padding: 8px 75px !important;
-    font-size: 24px !important;
+    padding: 8px 30px !important;
+    font-size: 16px !important;
     font-weight: lighter !important;
-}
\ No newline at end of file
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
May I know why 600px is chosen?

Reply:
600px is the sm breakpoint of material ui

Reply:
Ah I did not know that, thanks alot!